modules/core/jvm/src/main/scala/com/azavea/stac4s/StacItem.scala

```diff
@@ -29,8 +29,20 @@ final case class StacItem(
 
 object StacItem {
 
+  private val datetimeField: Lens[StacItem, ItemDatetime] = GenLens[StacItem](_.properties.datetime)
+
+  private val toDatetime = GenPrism[ItemDatetime, ItemDatetime.PointInTime]
+
+  private val toTimeRange = GenPrism[ItemDatetime, ItemDatetime.TimeRange]
+
   val propertiesExtension: Lens[StacItem, JsonObject] = GenLens[StacItem](_.properties.extensionFields)
 
+  val datetimePrism: Optional[StacItem, ItemDatetime.PointInTime] =
+    datetimeField.composePrism(toDatetime)
+
+  val timeRangePrism: Optional[StacItem, ItemDatetime.TimeRange] =
+    datetimeField.composePrism(toTimeRange)
```

Comment:
These are going to be really useful in Franklin when I have to update the `datetime`, `start_datetime`, and `end_datetime` columns -- the ADT and decoding will ensure that I only actually have one of these, so it will be safe to `getOption` using both prisms and insert all the values that come out of that in the db interaction